src/main/java/frc/robot/RobotContainer.java

```diff
@@ -84,6 +86,14 @@ private void configureBindings() {
   public Command getAutonomousCommand() {
     // An example command will be run in autonomous
     // return Autos.exampleAuto(m_exampleSubsystem);
-    return Commands.none();
+    return new AutoMobility(drive);
+  }
+
+  public void setDriveToBrake() {
+    drive.setDriveToBrake();
+  }
+
+  public void setDriveToCoast() {
+    drive.setDriveToCoast();
```

Comment:
in the future, just import the drivetrain object into Robot.java instead of doing this convoluted method